news/_posts/2019-05-23-announcing-end-of-support-for-obsolete-php-versions.md

```diff
@@ -0,0 +1,64 @@
+
+Doctrine is not the only library to stop supporting older PHP versions. Many other libraries like PHP-Parser and PhpUnit have dropped support for PHP 5 for a long time. Even some quality tools, like PhpStan, never supported it in the first place, preventing us from using them.
+
+In the case of Symfony, while they dropped support for PHP 5 on Symfony 4, PrestaShop is based on the 3.4 long-term-support (LTS) version, which is unlikely to drop support for PHP 5 in the near future.
+
+
+![Symfony version support roadmap](/assets/images/2019/05/symfony-support-roadmap.png){:.text-center}
+
+_Symfony version support roadmap. ([Source](https://symfony.com/roadmap#maintained-symfony-branches))_
+{:.text-center}
+
+Even though Symfony is the exception, the PHP dependency problem is becoming an increasingly complex problem, and we had to make a choice. Many other PHP-based CMS are facing the same problems: [Drupal dropped compatibility with PHP 5 back in March](https://www.drupal.org/node/2938726), while Wordpress, who has traditionally supported very old PHP versions, is now [requiring at least 5.6 and will soon drop it as well](https://wordpress.org/news/2019/04/minimum-php-version-update/).
+
+
+## Here is what we decided
+
+The upcoming PrestaShop 1.7.6.0 compatibility will stay unchanged. Starting from 1.7.7.0, and for every new minor or major version of PrestaShop, the PHP compatibility range will be decided as follows:
+
+*   **Lower bound:** the lowest version supported by our main dependencies, or the last version whose end-of-life date is no older than one year at the time of final release (whichever is highest).
+*   **Upper bound:** the latest stable version of PHP available at the time of that version's feature freeze, or the highest version supported by our main dependencies (whichever is the lowest).
+
+These rules will be applied in a “best effort” philosophy.
+
+For example, if the feature freeze date for 1.7.7 were to be set in December 2019, and the release in January 2020, then:
+
+**Lower bound:** PHP <= 7.0 would have been end-of-life for over a year, so those versions wouldn't be supported. PHP 7.1 would be end-of-life as well, but for less than a year, so it would qualify to become the lower bound — as long as our main dependencies support it.
+
+**Upper bound:** PHP 7.4 is due for release in December 2019, probably after our feature freeze date, so it is possible that not everything will be ready for it. In that case, the upper bound will be PHP 7.3.
+
+This would mean that PrestaShop 1.7.7 will most likely be compatible from PHP 7.1 to PHP 7.3.
+
+In conclusion, keeping up with newer PHP versions will not only allow developers to benefit from all the nice new language features, but it will also allow us to use newer, more recent libraries to make PrestaShop better and safer.
+
```

Comment:
It would be good to add a section "How to help", asking for reader to test PrestaShop with PHP 7.3 (and how?), and to open issues to identify the dependencies and the part of the project's own code to upgrade.
Build is for technical people, this is the place to ask this kind of help.

Reply:
Done